### PR TITLE
Use constant gmfDisplayQueryWindowOptions

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: contribs/gmf/src/options.js [hunk 1]
```diff
@@ -255,4 +255,12 @@ export function buildStyle(styleDescriptor) {
  * a single grid.
  */
 
+/**
+ * The display querry grid component options.
+ * @typedef {object} gmfDisplayQueryWindowOptions
+ * @property {Style} featuresStyle A style object for all features from the result of the query.
+ * @property {Style} selectedFeatureStyle A style object for the currently selected features.
+ * @property {boolean} [collapsed] f the query result window is collapsed.
```
Comment (ger-benjamin) [inline contribs/gmf/src/options.js:263]:
```suggestion
 * @property {boolean} [collapsed] If the query result window is collapsed.
```

File: contribs/gmf/examples/wfspermalink.html [hunk 2]
```diff
@@ -45,8 +45,7 @@
     </p>
 
     <gmf-displayquerywindow
-      gmf-displayquerywindow-desktop="false"
-      gmf-displayquerywindow-featuresstyle="ctrl.featureStyle">
+      gmf-displayquerywindow-desktop="false">
```
Comment (ger-benjamin) [inline contribs/gmf/examples/wfspermalink.html:48]:
```suggestion
      gmf-displayquerywindow-desktop="::false">
```
By the way

**Review by ger-benjamin — APPROVED** [review]
Looks good, two comments

File: contribs/gmf/examples/wfspermalink.html (hunk 2 above)
Comment (sbrunner) [reply contribs/gmf/examples/wfspermalink.html:48]: +1 :-)

**Pull request merged**